tool/site/client-react/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import {SummaryList} from "nhsuk-react-components"
+import * as React from "react"
+import {CommunicationRequest, CommunicationRequestPayload, MedicationRequest} from "fhir/r4"
+import {formatDate} from "../../formatters/dates"
+import {
+  getNumberOfRepeatsAllowedExtension,
+  getNumberOfRepeatsIssuedExtension, getPerformerSiteTypeExtension,
+  getRepeatInformationExtension
+} from "../../fhir/customExtensions"
+import {brLineFragments} from "./brLineFragments";
+
+export function createPrescriptionLevelDetails(medicationRequest: MedicationRequest, communicationRequests?: Array<CommunicationRequest>): PrescriptionLevelDetailsProps {
+  const prescriptionId = medicationRequest.groupIdentifier.value
+
+  const repeatExtension = getRepeatInformationExtension(medicationRequest.extension)
+  const repeatsIssuedExtension = getNumberOfRepeatsIssuedExtension(repeatExtension.extension)
```

Comment:
This code will fail if there is no repeat information extension.

Reply:
Also I think we should be using `MedicationRequest.dispenseRequest.numberOfRepeatsAllowed` rather than the extension now. Just watch out that numberOfRepeatsAllowed doesn't include the first issue because FHIR is cool